caphandler/endpoints.go

```diff
@@ -37,20 +37,21 @@ import (
 func GetEndpointCollection(ctx iris.Context) {
 	uri := ctx.Request().RequestURI
 	fabricID := ctx.Params().Get("id")
-	if _, err := capmodel.GetFabric(fabricID); err != nil {
+
+	endpointData, err := capmodel.GetAllEndpoints(fabricID)
+	if err != nil {
 		errMsg := fmt.Sprintf("failed to fetch endpoint data for uri %s: %s", uri, err.Error())
 		createDbErrResp(ctx, err, errMsg, []interface{}{"Endpoint", uri})
 		return
 	}
-	var members = []*model.Link{}
 
-	for endpointID, endpointData := range capdata.EndpointDataStore {
-		if endpointData.FabricID == fabricID {
-			members = append(members, &model.Link{
-				Oid: endpointID,
-			})
-		}
+	var members = []*model.Link{}
+	for endpointID := range endpointData {
```

Comment:
end point is not attached to a fabric anymore? But it should have a parent.

Reply:
Key set name used for storing address pool, zones and endpoints all have the fabric ID as the key, and all queries to fetch these data are using fabric ID.

Reply:
as above

Reply:
yes, multiple unique fabric IDs is supported